prbt_hardware_support/launch/system_info.launch

```diff
@@ -0,0 +1,20 @@
+<!--
```

Comment:
I think this file and the whole system info node could also be moved to the prbt_support package, because it only talks to the robot

Reply:
I like it.